### ISSUE TITLE
support `tea releases create` with asset upload

### ISSUE THREAD

**Issue by noerw** [go-gitea/tea#4]
I'm happy to provide a pull request implementing this functionality, but want to find consensus on the command line interface first.

I'd prefer a command like this:
```
tea releases create --tag v1.0 --target master -t v1.0 --asset artifact1.zip --asset artifact2
```

But I saw that in the HTTP API assets are handled in a separate endpoint. Would the API described above be fine anyway?

**Comment by lunny** [comment]
Please send your pull request, maintainers will review it.

**Comment by simbo1905** [comment]
The GitHub CLI [hub release](https://hub.github.com/hub-release.1.html) might provide some inspiration as to what the api could look like. For example you are suggesting creation a release from master which is a moving target; someone might push a broken commit just at the instance you are releasing. So the hub tool takes a “commitish” as the target such as a sha hash for a commit. By taking inspiration from that mature tool it will make it easier for people to write release automation scripts that are easy to migrate from other git servers onto gitea.

**Comment by noerw** [comment]
was implemented with #6
